OsmAnd-java/src/main/java/net/osmand/util/OpeningHoursParser.java

```diff
@@ -33,12 +39,16 @@
 	private static final int WITHOUT_TIME_LIMIT = -1;
 	private static final int CURRENT_DAY_TIME_LIMIT = -2;
 
+	private static boolean twelveHourFormatting;
+	private static DateFormat twelveHourFormatter;
+	private static DateFormat twelveHourFormatterAmPm;
+
 	static {
 		DateFormatSymbols dateFormatSymbols = DateFormatSymbols.getInstance(Locale.US);
 		monthsStr = dateFormatSymbols.getShortMonths();
 		daysStr = getLettersStringArray(dateFormatSymbols.getShortWeekdays(), 2);
 
-		initLocalStrings();
```

Comment:
Suggestion: leave `initLocalStrings` as it is and add `static setTwelveHourFormattingEnabled(boolean enabled)`